### ISSUE TITLE
Common Movement scripts

### ISSUE THREAD

**Issue by m-jojo-s** [MahadAdnanGul/WinterGameJam#15]
For the bots and players, can we keep a common movement script? Since we're not using forces directly for movement, affecting movement from external objects like walls will become difficult to manage if we add more such functionality. Otherwise we'll need to refer both movement scripts every time and sync changes to both of them as well.

**Comment by MahadAdnanGul** [comment]
I agree we should, The same script can diverge a bit if the game object it is attached to is a bot

**Comment by ZuhairGhias** [comment]
Yea I can take a look
